tests/test_bincopy.py

```diff
@@ -92,6 +92,90 @@ def test_bad_srec(self):
+            binfile.add_ti_txt(fin.read())
+
+        with open('tests/files/empty_main.bin', 'rb') as fin:
+            self.assertEqual(binfile.as_binary(padding=b'\x00'), fin.read())
+
+        # Add and overwrite the data.
+        binfile = bincopy.BinFile()
+        binfile.add_ti_txt_file('tests/files/empty_main_rearranged.s19.txt')
+        binfile.add_ti_txt_file('tests/files/empty_main_rearranged.s19.txt',
+                              overwrite=True)
+
+        with open('tests/files/empty_main.bin', 'rb') as fin:
+            self.assertEqual(binfile.as_binary(padding=b'\x00'), fin.read())
+
+        empty = bincopy.BinFile()
+        binfile = bincopy.BinFile('tests/files/empty.txt')
+        self.assertEqual(binfile.as_ti_txt(), empty.as_ti_txt())
+
+    def test_bad_ti_txt(self):
+        cases = (('tests/files/bad_ti_txt_address_value.txt', 'bad address'),
+                 ('tests/files/bad_ti_txt_bad_q.txt',         'bad file terminator'),
+                 ('tests/files/bad_ti_txt_data_value.txt',    'bad data'),
+                 ('tests/files/bad_ti_txt_length.txt',        'bad record length'),
+                 ('tests/files/bad_ti_txt_no_offset.txt',     'missing offset'),
+                 ('tests/files/bad_ti_txt_no_q.txt',          'missing file terminator'))
+
+        for filename, message in cases:
+            binfile = bincopy.BinFile()
+            with self.assertRaises(bincopy.Error) as cm:
+                binfile.add_ti_txt_file(filename)
+
+            self.assertIn(message, str(cm.exception), 
+                          "When parsing {}, expected exception with \"{}\" instead of \"{}\"".format(filename, message, str(cm.exception)))
+
+    def test_compare_ti_txt(self):
+        pairs = (("tests/files/in.s19",
+                  "tests/files/in.s19.txt"),
+                 ("tests/files/in.hex",
+                  "tests/files/in.hex.txt"),
+                 ("tests/files/empty_main.s19",
+                  "tests/files/empty_main.s19.txt"),
+                 ("tests/files/convert.s19",
+                  "tests/files/convert.s19.txt"),
+                 ("tests/files/out.s19",
+                  "tests/files/out.s19.txt"),
+                 ("tests/files/non_sorted_segments.s19",
+                  "tests/files/non_sorted_segments.s19.txt"),
+                 ("tests/files/non_sorted_segments_merged_and_sorted.s19",
+                  "tests/files/non_sorted_segments_merged_and_sorted.s19.txt"),
+
+                 ("tests/files/in.hex",
+                  "tests/files/in.hex.txt"),
+                 ("tests/files/in.hex",
+                  "tests/files/in.hex.txt"),
+                 ("tests/files/empty_main.hex",
+                  "tests/files/empty_main.hex.txt"),
+                 ("tests/files/convert.hex",
+                  "tests/files/convert.hex.txt"),
+                 ("tests/files/out.hex",
+                  "tests/files/out.hex.txt"))
```

Comment:
I suggest adding the `tests/files/` prefix in the loop below, making the list of test files easier to read.